mypy/nodes.py

```diff
@@ -117,13 +116,21 @@ def accept(self, visitor: NodeVisitor[T]) -> T:
         raise RuntimeError('Not implemented')
 
 
-# These are placeholders for a future refactoring; see #1783.
-# For now they serve as (unchecked) documentation of what various
+# This is a placeholder for a future refactoring; see #1783.
+# For now it serves as (unchecked) documentation of what various
 # fields of Node subtypes are expected to contain.
-Statement = Node
 Expression = Node
 
 
+class Statement(Node):
+    pass
+
+
+# For now, only Var is a declaration
```

Comment:
I am not sure it's needed; the motivation was that `Var` is not a category like `Statement` and `Expression`. But now I see that `Argument` should be together with `Var` too (wherever that might be) and PEP-526 might add more.

It feels like it's appropriate here, but I don't have a real Argument to support this feeling.